.azure-pipelines/integration-tests.yml

```diff
@@ -68,10 +69,22 @@ jobs:
 
   steps:
   - task: UseDotNet@2
-    displayName: install dotnet core 2.1 sdk
+    displayName: install dotnet core runtime 2.1
+    inputs:
+      packageType: runtime
+      version: 2.1.x
+
+  - task: UseDotNet@2
```

Comment:
We should try to remove sample apps that are on .NET Core 3.0. Not sure if we want to do that in this PR but it was something worth mentioning

Reply:
Whoops, just realized one of the two samples you're adding is for AspNetCore 3.0. Disregard this.